agentlocal/agent_local.go

```diff
@@ -63,6 +63,11 @@ type Server struct {
 	reloadCloseOnce sync.Once
 }
 
+func (s *Server) mustEmbedUnimplementedAgentLocalServer() {
```

Comment:
it's wrong, possible solution remove this requirement by some protoc --arg or by embeding [agentlocalpb|agentpb].UnimplementedAgentLocalServer